### ISSUE TITLE
Silly Suggestion: Later Game Achivements

### ISSUE THREAD

**Issue by Teirdalin** [GTNewHorizons/GT-New-Horizons-Modpack#3693]
#### Which modpack version are you using?
2.0.5.1
#
#### What do you mean later game achievements?

This is a bit pointless of a thing to suggest, but I think later game could use some more achievements for things.     Like making the neutronium compressor and getting your first infinity.   Making the draconic gear and tools.   And numerous other things.

**Comment by KaneHart** [comment]
Silly or not, I kind of wish we had achievements / trophies.

**Comment by Dream-Master** [comment]
@KaneHart sure if you have time code it :) I am not against it but literally haven't the time.

**Comment by KaneHart** [comment]
Yeah 100% I can always look into it. Not sure if I could code one, but I do know one the FTB packs had something cool I thought was in 1.7.10, but I will keep my eye out. Something to celebrating Tiers and Big Accomplishments.

**Comment by glowredman** [comment]
I've looked a bit into this because I'd like to have something like trophies implemented aswell. @KaneHart is right, FTB Infinity Evolved Skyblock has custom achievements:
![grafik](https://user-images.githubusercontent.com/35727266/122944470-2417ae00-d378-11eb-9340-28fcd85a2629.png)

 When the player gets one of these achievements they get a trophy aswell:
![grafik](https://user-images.githubusercontent.com/35727266/122945482-e6ffeb80-d378-11eb-934f-2c7c69616fdd.png)

**How it works:**
The achievements are added via [CutomThings](https://www.curseforge.com/minecraft/mc-mods/customthings). This mod rewards the player with an item upon completing an achievement, in this case a trophy from [FTB Trophies](https://www.curseforge.com/minecraft/mc-mods/ftbtrophies). The trophies are defined in the FTBTrophies jar under `assets/ftbtrophies/models/FTBTrophies/201604031752_FTBTrophies.json` using the models in `assets/ftbtrophies/models/FTBTrophies/201604031752_FTBTrophies.cspack`. The models are made using [CraftStudio](https://sparklinlabs.itch.io/craftstudio). FTB Trophies depends on PTRModelLib to render the trophies which is included in Decocraft (Decocraft is mentioned as required in the @Mod-annotation in FTB Trophies aswell).

**What can be done:**
Adding achievements for major accomplishments isn't hard and could be done without any additional mods (e.g. in GTNH Core).
If we want to go a step further and add trophies, there (at least) are these options:
- Make a completely custom implementation for trophies (via GTNH Core or a new mod)
Pros: complete control over content and licensing; least amount of bloat
Cons: huge amount of work
- Ask RazzleberryFox/LatvianModder for permissions to redistribute modified versions of FTB Trophies. They might be inclined to do so because there is a repository named "[FTB-Trophies](https://github.com/FTBTeam/FTB-Trophies)" owned by the FTBTeam, although it contains only a README. We would be able to remove Decocraft as a hard-dependency and replace the trophy-models. This would require permissions (ask RazzleberryFox/profmobius) to use PTRModelLib outside of Decocraft. For MC1.12.2 PTRModelLib is published seperately on [CurseForge](https://www.curseforge.com/minecraft/mc-mods/ptrlib) and [profmobius' maven](https://www.mobiusstrip.eu/maven/com/mia/craftstudio/PTRLib/), so they might be inclined to publish it for 1.7.10 aswell or at least allow us to use it without Decocraft.
Pros: minimal work
Cons: original authors might not replay or decline
- Add both FTB Trophies and Decocraft to the pack and override the FTB Trophies models using ResourceLoader (already included in the pack, used for CustomMainMenu)
Pros: minimal work
Cons: both FTB Trophies and Decocraft are ARR; we might not want to add Decocraft due to bloat/performance/whatever reasons
- Add both FTB Trophies and Decocraft to the pack, override the models and somehow hack into Decocraft to remove any content
Pros: less work than first option
Cons: I don't know how mush work it'd be to remove/disable any Decocraft content
- If we only get permissions to redistribute PTRModelLib and no permissions to edit&redistribute FTB Trophies, we might still be able to use FTB Trophies by fooling it to think Decocraft is present (either by forcing Forge to ignore that FTB Trophies misses a dependency or by creating an "empty" mod with the same modid as Decocraft). Thus, we'd be able to override the models using ResourceLoader
Pros: easy solution if we don't get permissions for FTB Trophies
Cons: FTB Trophies would still be ARR

If we want to add trophies in any form, we need someone to model them

Note: PTRModelLib is called PTRLib outside it's sourcecode (don't ask me why)

**Comment by Prometheus0000** [comment]
Decocraft was already decided to not be added.

**Comment by KaneHart** [comment]
bot needs to die
